components/form/pta/src/index.js

```diff
@@ -0,0 +1,74 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+
+export const BASE_CLASS = 'sui-FormPta'
+
+class FormPta extends Component {
+  /**
+   * Avoid iframe re-rendering
+   */
+  shouldComponentUpdate () {
+    return false
+  }
+
+  /**
+   * Send settings to iframe content form
+   */
+  sendSettingsToForm = ({ target: { contentWindow } }) => {
+    const { redirectOnErrorUrl, formUrl, enableDraft, redirectOnSuccessUrl } = this.props
+    const formSettings = {
+      enableDraft,
+      formUrl,
+      exitURLs: {
+        success: redirectOnSuccessUrl,
+        error: redirectOnErrorUrl
+      }
+    }
+
+    contentWindow.postMessage(formSettings, '*')
+  }
+
+  render () {
+    const { props: { formUrl }, sendSettingsToForm } = this
+
+    return (
+      <div>
+        <h1>PTA Form builder</h1>
```

Comment:
are you really displaying this title in ALL sites ? No translation ?
I thought it was going to be only an iframe.

Reply:
Upss, you are right!

Reply:
As David said. Is it going to be an iframe wrapper? or are you going to add the title too?